scipy/stats/morestats.py

```diff
@@ -3504,5 +3505,6 @@ def circstd(samples, high=2*pi, low=0, axis=None, nan_policy='propagate'):
         sin_mean = sin_samp.sum(axis=axis) / nsum
         cos_mean = cos_samp.sum(axis=axis) / nsum
     R = hypot(sin_mean, cos_mean)
+    R = np.minimum(R, 1)  # hypot can go slightly above 1 due to rounding errors
```

Comment:
slightly cleaner to avoid reassignment of R?

```python
# NOTE: hypot can go slightly above 1 due to rounding errors
R = np.minimum(hypot(sin_mean, cos_mean), 1) 
```